openpgp.d.ts

```diff
@@ -98,6 +98,10 @@ export class Subkey {
   public getCreationTime(): Date;
   public getAlgorithmInfo(): AlgorithmInfo;
   public getKeyID(): KeyID;
+  public getExpirationTime(date?: Date, config?: Config): Promise<Date | typeof Infinity | null>
+  public clone(): Subkey;
```

Comment:
```suggestion
```
Not sure about exposing this one. Do you have a use case for it?

Reply:
Actually I don't, I'm used to work with typescript and assumed that the method would have been marked private if it wasn't used outside the class. But this is only available in typescript file and not in javascript my bad. I added a commit to remove this line.